admin/src/views/UserSearch.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <label>Usersuche</label>
```

Comment:
localise

Reply:
ist auf dem schirm, wird zu einem späteren zeitpunkt in einem abwasch gemacht.